src/inmanta/db/versions/v202212090.py

```diff
@@ -0,0 +1,34 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+
+async def update(connection: Connection) -> None:
+    await connection.execute(
+        """
+-- adds the environment to which a metric is related
+ALTER TABLE public.environmentmetricsgauge ADD COLUMN environment uuid NOT NULL REFERENCES environment(id) ON DELETE CASCADE;
```

Comment:
did we already release the previous version? if we didn't I would just update that one?

Reply:
Indeed, that would have been easier, it is changed.